doc/user-guide/io.rst

```diff
@@ -984,6 +984,42 @@ reads. Because this fall-back option is so much slower, xarray issues a
     3. Explicitly setting ``consolidated=True``, to raise an error in this case
        instead of falling back to try reading non-consolidated metadata.
 
+.. _io.kerchunk:
+
+Kerchunk
+--------
+
+`Kerchunk <https://fsspec.github.io/kerchunk/index.html>`_ is a Python library that allows you to access chunked and compressed data formats (such as NetCDF3, NetCDF4, HDF5, GRIB2, TIFF & FITS), many of which are primary data formats for many data archives, as if they were in ARCO formats such as `Zarr`_ which allows for parallel, chunk-specific access. Instead of creating a new copy of the dataset in the Zarr spec/format, Kerchunk reads through the data archive and extracts the byte range and compression information of each chunk, then writes that information to a ``.json`` file, which is later used as an ephemeral ``zarr`` dataset.
```

Comment:
I think some clearer distinction between kerchunk the library and the kerchunk format for references would be useful. Especially now that there are multiple libraries that can create the kerchunk-formatted references. 

I would have structured it more like:

"Kerchunk reference files are a metadata format that store the paths and locations of byte ranges of chunks in other files, e.g. netCDF, ...

These reference files can be created with the kerchunk or [virtualizarr](https://github.com/zarr-developers/VirtualiZarr) libraries - see the documentation of those packages for details.

Once created, xarray can open these references to read data form the original files using the `kerchunk` backend, like this...
"

Reply:
Well the engine is called "kerchunk" :). Actually, "referenceFS" might have been more accurate, following your comment - but never mind. I think mentioning virtualizarr here is reasonable.

Separately: @TomNicholas , we at some point mentioned maybe folding virtualizarr ("vz" I'll call it from now on) into the kerchunk codebase. Is that something you want to do now or in the future?

Reply:
Virtualizarr follows kerchunk's reference format?

Reply:
vz implements an alternative way to do what MultiZarrToZarr does, and produces reference sets that are (can be) compatible. So engine="kerchunk" can be used to open vz's output.

@TomNicholas , when do you think it will be appropriate to advertise engine="virtualizarr" here too?

Reply:
> Well the engine is called "kerchunk" :). Actually, "referenceFS" might have been more accurate, following your comment - but never mind.

I still think titling this section "Kerchunk" is appropriate! We're talking about how to open files that follow the [kerchunk references specification](https://fsspec.github.io/kerchunk/spec.html). I just think we can also mention that there is now more than one way to create the references files.

> vz implements an alternative way to do what MultiZarrToZarr does, and produces reference sets that are (can be) compatible. So engine="kerchunk" can be used to open vz's output.

Exactly. The reference sets created should be basically identical (if they aren't then its a bug).

> @TomNicholas , when do you think it will be appropriate to advertise `engine="virtualizarr"` here too?

I think it's ready to advertise, it's not feature complete but its stable and working. I mostly just need to make sure its up on PyPI and conda-forge. But if other people think we should wait until its a bit more battle-tested then I don't mind deferring the addition to a later docs PR.

> Separately: @TomNicholas , we at some point mentioned maybe folding virtualizarr ("vz" I'll call it from now on) into the kerchunk codebase. Is that something you want to do now or in the future?

That was my original intention, but now I think it might be better for the codebases to stay separate. It's working well for virtualizarr to simply import just the kerchunk file reader backends. I wrote out a more detailed [rationale of this decision here](https://virtualizarr.readthedocs.io/en/latest/faq.html#why-a-new-project). Also we've now moved virtualizarr into the zarr-developers github org, which I think is a great place for it considering the intended future development (i.e. greater integration with zarr-python itself). 

> "vz" I'll call it from now on

I can't believe it never occurred to me to abbreviate it to that - maybe I'll start doing `import virtualizarr as vz` from now on 😁

Reply:
> I can't believe it never occurred to me to abbreviate it to that

@TomNicholas way ahead of you, I've been using `import earthaccess.virtualizarr as evz`